documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorage.java

```diff
@@ -1106,11 +1106,11 @@ public void updatePermissions(String documentID, NodePermission nodePermissionEn
       if (node.canAddMixin(NodeTypeConstants.EXO_PRIVILEGEABLE)) {
         node.addMixin(NodeTypeConstants.EXO_PRIVILEGEABLE);
       }
+      Calendar now = Calendar.getInstance();
+      node.setProperty(NodeTypeConstants.EXO_DATE_MODIFIED, now);
+      node.setProperty(NodeTypeConstants.EXO_LAST_MODIFIED_DATE, now);
+      node.save();
```

Comment:
The issue was that the changes contain opposite instructions, that means the user removes possibility to update file and tries to update the lastModification date in the same time which will execute the permission removal , then it will try to update the modification date.
Removing this block should fix the issue. could you test that ?